File: exam20210318_constraintlayout/app/src/main/java/com/test/exam20210318_constraintlayout/MainActivity.java
package com.test.exam20210318_constraintlayout;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        //activity_main 이였던 xml을 kakao_login 으로 변경
        //setContentView(R.layout.activity_main);
        //setContentView(R.layout.kakao_login);
        setContentView(R.layout.kakao_login_form);
    }
}

File: exam01/app/src/main/java/com/test/exam01/MainActivity.kt
package com.test.exam01

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.ImageView
import android.widget.TextView

public class MainActivity extend AppCompatActivity implements View.OnClickListner{
    Button btn_shake;
    ImageView img1, img2;
    TextView tv_score;

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


    }
}